spec.html

```diff
@@ -28583,7 +28588,7 @@ <h1>TimeClip ( _time_ )</h1>
         <p>The abstract operation TimeClip takes argument _time_ (a Number). It calculates a number of milliseconds. It performs the following steps when called:</p>
         <emu-alg>
           1. If _time_ is not finite, return *NaN*.
-          1. If abs(_time_) &gt; 8.64 &times; 10<sup>15</sup>, return *NaN*.
+          1. If abs(_time_) &gt; 8.64<sub>𝔽</sub> &times; 10<sub>𝔽</sub><sup>15<sub>𝔽</sub></sup>, return *NaN*.
```

Comment:
Exponentiation is not defined on Numbers. And it's weird to invoke it here using a Number exponent.

Reply:
What does `Math.pow`/`**` do on Numbers?